Orm/Xtensive.Orm/Orm/Rse/Providers/Compilable/OrderProviderBase.cs

```diff
@@ -66,11 +66,10 @@ protected override void Initialize()
       var comparisonRules = new ComparisonRules[Order.Count];
       for (int i = 0; i < Order.Count; i++) {
         var orderItem = Order[i];
-        var culture = CultureInfo.InvariantCulture;
         var column = Header.Columns[orderItem.Key];
-        var mappedColumn = column as MappedColumn;
-        if (mappedColumn != null && mappedColumn.ColumnInfoRef != null)
-          culture = mappedColumn.ColumnInfoRef.CultureInfo;
+        var culture = column is MappedColumn mappedColumn && mappedColumn.ColumnInfoRef != default
```

Comment:
Am I right this creates memory traffic by making copies of ColumnInfoRef to send it to the inequality operator and then to the Equals method?

Reply:
Yes, you are right
optimized it